tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: add-fbc-contribution
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to create internalrequests to add contributions to index images
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+      default: "snapshot_spec.json"
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+      default: "data.json"
+    - name: binaryImage
+      type: string
+      description: binaryImage value updated by update-ocp-tag task
+    - name: fromIndex
+      type: string
+      description: fromIndex value updated by update-ocp-tag task
+    - name: requestTimeout
+      type: string
+      default: "180"
+      description: InternalRequest timeout
+  results:
+    - name: fbcOptIn
+      description: Result to set weather the contribution is FBC opt in
+    - name: requestResultsFile
+      description: Internal Request results file
+    - name: requestMessage
+      description: Internal Request message
+    - name: requestReason
+      description: Internal Request reason
+  workspaces:
+    - name: data
+      description: workspace to read and save files
+  steps:
+    - name: add-contribution
+      image: quay.io/hacbs-release/release-utils:a9603c0ae1982be5e1142d44765b1475adc7ad40
+      script: |
+        #!/usr/bin/env sh
+        #
+        set -e
+
+        SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No valid data file was provided."
+            exit 1
+        fi
+
+        echo -n "$(workspaces.data.path)/ir-$(context.taskRun.uid)-result.json" > $(results.requestResultsFile.path)
```

Comment:
This was changed to `.json`, but down below on line 91 the script's output is still saved to a `.txt` file.

Reply:
They are actually 2 different things. One is just to get the command output, and the .json is to save the IIB result. The difference is beforehand I was reusing the file. Maybe I should change the `.txt`  to `.log`?

Reply:
Ah, ok, I didn't realize they were two different files. I agree it's better to have different names.